monkey/infection_monkey/network/ping_scanner.py

```diff
@@ -59,10 +59,10 @@ def get_host_fingerprint(self, host):
         if regex_result:
             try:
                 ttl = int(regex_result.group(0))
-                if LINUX_TTL == ttl:
-                    host.os['type'] = 'linux'
-                elif WINDOWS_TTL == ttl:
+                if (ttl > LINUX_TTL) and (ttl <= WINDOWS_TTL):
```

Comment:
If you think it's better to check the whole range of TTL it would make sence to change WINDOWS_TTL to MAX_TTL and set it to 255

Reply:
Wouldn't it make it less clear why I mark it as Windows if it's in that range?

Reply:
Yes, I didn't provide a good solution. Now I think we should just make it like so:
if ttl <= LINUX_TTL:
     host.os['type'] = 'linux'
else:
     host.os['type'] = 'windows'
It's simple and includes whole range.